README.md

```diff
@@ -0,0 +1 @@
+# Docker Compose
```

Comment:
I'll approve this for now, but we need to add more significant details about the project, requirements and how to run it.